src/graphics/tessellation.rs

```diff
@@ -52,64 +56,53 @@ fn build_path(points: &[Point], closed: bool) -> path::Path {
     path_builder.build()
 }
 
-type BuffersBuilder<'a> = geometry_builder::BuffersBuilder<'a, Vertex, math::Point, ScreenUV>;
-
-fn build_geometry<F>(f: F) -> GameResult<Buffer>
-    where F: for<'a> FnOnce(&mut BuffersBuilder<'a>)
-                            -> Result<tessellation::geometry_builder::Count, ()>
+fn build_geometry<F, V, E>(line_width: f32, f: F) -> GameResult<Buffer>
+    where F: for<'a> FnOnce(&mut geometry_builder::BuffersBuilder<'a, Vertex, V, VertexConstructor>)
+                            -> Result<geometry_builder::Count, E>,
+        VertexConstructor: geometry_builder::VertexConstructor<V, Vertex>,
 {
     let mut buffers = geometry_builder::VertexBuffers::new();
+    let vertex_ctor = VertexConstructor { stroke_width: line_width };
     {
-        let mut builder = geometry_builder::BuffersBuilder::new(&mut buffers, ScreenUV);
-        if let Err(()) = f(&mut builder) {
+        let mut builder = geometry_builder::BuffersBuilder::new(&mut buffers, vertex_ctor);
+        if let Err(_) = f(&mut builder) {
             return Err(GameError::RenderError(String::from("geometry tessellation failed")));
         }
     }
     Ok(buffers)
 }
 
-pub fn build_line(points: &[Point], line_width: f32) -> GameResult<Buffer> {
-    let path = build_path(points, false);
-    let opts = path_stroke::StrokeOptions::stroke_width(line_width);
+fn build_stroke(points: &[Points], close: bool, line_width: f32) -> GameResult<Buffer> {
```

Comment:
Yes, this should have been fixed yesterday I think. Sorry for pushing bad code. The current CI failure is only on beta and was caused by a regression: https://github.com/rust-lang/rust/issues/41944